.github/workflows/sonarqube.yml

```diff
@@ -0,0 +1,38 @@
+name: SonarQube
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+
+jobs:
+  main:
+    runs-on: zendesk-stable
```

Comment:
I think here you'd be after `runs-on: [self-hosted, zendesk-stable]`.

Reply:
There might be a couple of different syntaxes that work here, I've used this format with success before on [Apps Asset Service](https://github.com/zendesk/apps_asset_service/pull/17/files#diff-a5e11e7bedd328777f4897395a711a4492d413b70b12479230470ec608d96792R11) and [Developer Docs](https://github.com/zendesk/developer-docs/pull/141/files#diff-a5e11e7bedd328777f4897395a711a4492d413b70b12479230470ec608d96792R11)

Reply:
Actually sorry I think you're right Chris. Should I update those two repos to be `[self-hosted, zendesk-stable]` as well?

Reply:
Turns out as a public repo this needs to use a Github-hosted runner rather than one of our self-hosted ones